fastlane/Fastfile

```diff
@@ -127,6 +129,31 @@ platform :ios do
         build_and_upload_app(options)
     end
 
+    lane :trigger_api_build_cirrus_ci do |options|
+        authorization_header = { "Authorization" => "Bearer #{ENV['CIRRUS_CI_API_TOKEN']}" }
+        uri = 'https://api.cirrus-ci.com/graphql'
+        create_build = HTTParty.post(
+            uri,
+            body: "{ \"query\": \"mutation CreateBuildMutation($input: RepositoryCreateBuildInput!) { createBuild(input: $input) { build { id } } }\", \"variables\": { \"input\": {\"repositoryId\": \"4507403593449472\", \"branch\": \"#{ENV['GITHUB_REF_NAME']}\", \"clientMutationId\": \"trigger-build-on-cirrus\"}} }",
+            headers: authorization_header
+        )
+        created_build_id =  JSON.parse(create_build.body, symbolize_names: true)[:data][:createBuild][:build][:id]
+        `sleep 7`
```

Comment:
feels a bit hacky

Reply:
Yeah, but it seems there is no other option rather than creating a websocket and listening for changes which for me seems to be to complicated for that case

Reply:
sorry I still don't get why we have a sleep of 7?

Reply:
Seems that scheduling jobs for the newly created pipeline is an async event (which seems reasonable actually to process .cirrus.yml for all possible jobs that may be triggered by the API event)
So I preferred to wait a bit longer than 1 second (that would be probably enough for our repo) since the whole GH action would take less than a minute

Reply:
can you add a comment then to it so we don't forget in 6 months?